src/assets/logo/Logo.jsx

```diff
@@ -0,0 +1,5 @@
+import logo from "./logo__image.png";
+
+export const Logo = ({ classes }) => {
```

Comment:
`Logo` **image file** should be in assets and it should be imported to the `Logo` **component file** which should be in components.